### PR TITLE
feat: close RightDrawer with ESC key

### LINKED ISSUE #576: AAU I can close right drawer with esc key

### PR DESCRIPTION
**Pull request by thaisguigon**
Closes #576

### CODE REVIEW

File: front/src/modules/ui/right-drawer/components/RightDrawer.tsx [hunk 1]
```diff
@@ -47,6 +47,17 @@ export function RightDrawer() {
     mode: OutsideClickAlerterMode.absolute,
   });
   const theme = useTheme();
+
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
```
Comment (emilienchvt) [inline front/src/modules/ui/right-drawer/components/RightDrawer.tsx:52]: We have a `useScopedHotkeys` hook already implemented for this kind of use case: https://github.com/twentyhq/twenty/blob/8b7314cd39100d06e001ef602ced5a403d535ac4/front/src/modules/ui/hotkey/hooks/useScopedHotkeys.ts#L13

An example of implementation can be found here:
https://github.com/twentyhq/twenty/blob/8b7314cd39100d06e001ef602ced5a403d535ac4/front/src/modules/ui/filter-n-sort/components/FilterDropdownButton.tsx#L86-L93

Comment (charlesBochet) [reply front/src/modules/ui/right-drawer/components/RightDrawer.tsx:52]: +1, we should use useScopedHotkeys and use RightDrawer scope!
Sorry @thaisguigon, you could not invent it! I'm starting documenting things on our documentation so it's easier for new joiners

**Review by charlesBochet — CHANGES_REQUESTED** [review]

**Review by charlesBochet — APPROVED** [review]

**Review by charlesBochet — COMMENTED** [review]
use useScopedHotkeys (see comments above)

File: front/src/modules/ui/right-drawer/components/RightDrawer.tsx (hunk 1 above)
Comment (thaisguigon) [reply front/src/modules/ui/right-drawer/components/RightDrawer.tsx]: Ok, done!

**Pull request merged**